Sources/SafeDICore/Models/List.swift

```diff
@@ -26,6 +27,8 @@ public final class List<Element>: Sequence {
         self.value = value
         self.previous = previous
         self.next = next
+        previous?.next = self
+        next?.previous = self
```

Comment:
now our initializer makes suer the `previous` and `next` nodes aren't a lie.

a little odd to have an initializer side effect on other instances... but without it this initializer can leave us in a pretty bad place.